DOCUMENTATION.md

```diff
@@ -244,7 +251,7 @@ You can leave out this argument if your service only has one reply-to email addr
 
 ### Response
 
-If the request to the client is successful, the promise resolves with an `object`:
+If the request is successful, the promise resolves with a `response` `object`, such that `response.data` is:
```

Comment:
Is 'such that' a specific piece of developer language in this context, or can we say this another way?